### DISCUSSION TITLE
ServiceStack.Redis issue writing to multiple redis

### DISCUSSION THREAD

**Discussion by wurangbo** [ServiceStack/Discuss#128] [Q&A]
hi  Guys,
When I use ServiceStack.redis to write to multiple redis using the following configuration I encounter the following problem：

<add key="RealRedisServer" value="siteweb1!@192.168.4.40:6479,siteweb1!@192.168.4.41:6479,siteweb1!@192.168.2.228:6479"/>
<add key="RealReadRedisServer" value="siteweb1!@192.168.4.40:6479,siteweb1!@192.168.4.41:6479,siteweb1!@192.168.2.228:6479"/>
<add key="RealWriteRedisServer" value="siteweb1!@192.168.4.40:6479,siteweb1!@192.168.4.41:6479,siteweb1!@192.168.2.228:6479"/>

Objects can only be written to ：siteweb1!@192.168.4.40:6479,siteweb1!@192.168.4.41:6479
siteweb1!@192.168.2.228:6479 Write did not take effect。

**Comment by Layoric** [comment]
I don't understand the configuration you've provided, how are you using that string to connect using ServiceStack.Redis connection pool? It looks like 3 of the same server and port, is this a Redis Sentinel setup or something else? See [here for usage of Redis Sentinel connection](https://docs.servicestack.net/redis/sentinel#usage). I'll need more context to how you are using this string to connect to redis to help much further, provide as much code as you can.

**Reply by wurangbo** [reply]
<add key="RealRedisServer" value="siteweb1!@192.168.4.40:6479,siteweb1!@192.168.4.41:6479,siteweb1!@192.168.2.228:6479"/>
Sorry, I updated the issue description。
